tests/filter_test.php

```diff
@@ -104,26 +107,26 @@ public function test_image_opt_url() {
         set_config('maxwidth', '480', 'filter_imageopt');
 
         $fixturefile = 'testpng_2880x1800.png';
-        $context = context_system::instance();
+        $context = \context_system::instance();
```

Comment:
Tests aren't namespaced and therefore you don't need to add a backslash before the classes.

Reply:
This goes for every other inserted backslash before a class in this file.

Reply:
AH - I am wrong. I see that you have inserted a namespace at the top of the file. Please try to add use statements to avoid repeated backslashing.

E.g. after "defined('MOODLE_INTERNAL') || die();"

```
use context_system
```